.eslintrc.js

```diff
@@ -0,0 +1,44 @@
+module.exports = {
+    "extends": [
+	"eslint:recommended",
```

Comment:
The indentation in this file is a mess, it mixes 3-spaces, 4-spaces and tabs. :/

Reply:
ill fix